docs/examples/basic_conceptual_guide/Part_3-model_analyzer/Readme.md

```diff
@@ -0,0 +1,108 @@
+
+With Model Analyzer users can: 
+* Run customizable configuration sweeps can be done to identify the best possible configuration for the expected workload and hardware. 
+* Summarize findings about latency, throughput, GPU resource utilizations, power draw and more, with detailed reports, metrics and graphs. These reports help compare performance across different configurations of setup.
+* Tailor model deployments to cater to user's Quality of Service requirements like specific p99 latency limits, GPU memory utilization and more!
+
+## Using Model Analyzer
+ 
+### Pre-requisites
+
+Refer to Part 2 of this series to get access to the models. Refer to the Model Analyzer [installation guide](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/install.md#recommended-installation-method) for more information about installing Model Analyzer. For ease of following along, use these commands to install model analyzer:
+
+```
+sudo apt-get update && sudo apt-get install python3-pip
+sudo apt-get update && sudo apt-get install wkhtmltopdf
+pip3 install triton-model-analyzer
+```
+
+### Usage details
+
+Before diving into details with an example, a discussion about the overall functionality and capability is needed to understand how best to use the Model Analyzer tool. Let's begin the discussion by focusing on what is most important to the users: setting `objectives` and `constraints` for the sweeps.
+
+- **objectives**: Users can choose to order the results on the basis of their deployment goals, throughput, latency, or tailoring to specific resource constraints. [Learn more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#objective).
+    
+    Model Analyzer has two modes, Online and Offline. In online mode users can specify latency budgets for their deployments to cater to their requirements. For Offline mode a similar specification can be mode for minimum throughput. [Learn more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/cli.md#model-analyze-modes)
+
+- **constraints**: Users can also choose to constrain the selection of sweeps to specific requirements for throughput, latency or gpu memory utilization. [Learn more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#constraint)
+
+With the more broader selections discussed, let's talk about the three key sub commands that users need to use to make use of Model Analyzer: `profile`, `analyze`, and `report`. While most of the settings for these commands can be set using a flag, some require building a configuration file. Refer to [this section](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md) of the documentation for a comprehensive list of settings.
```

Comment:
There are just two steps now: Profile and Report. (Analyze has been folded into profile)